docs/source/api/v4/servers.rst

```diff
@@ -159,6 +164,8 @@ Response Structure
 :updPending: A boolean value which, if ``true``, indicates that the server has updates of some kind pending, typically to be acted upon by Traffic Control Cache Config (:term:`t3c`, formerly ORT)
 :xmppId:     A system-generated UUID used to generate a server hashId for use in Traffic Router's consistent hashing algorithm. This value is set when a server is created and cannot be changed afterwards.
 :xmppPasswd: The password used in XMPP communications with the server
+:asns:       The :abbr:`ASN (Autonomous System Number)` associated with the cachegroups of the current server.
```

Comment:
All of the other fields are in alphabetical order. This should be at the top of the list.